lib/GADS.pm

```diff
@@ -1857,6 +1858,7 @@ any qr{/(record|history|purge|purgehistory)/([0-9]+)} => require_login sub {
     my ($return, $options, $is_raw) = _process_edit($id, $record);
     return $return if $is_raw;
     $return->{is_history} = $action eq 'history';
+    $return->{reports} = schema->resultset('Report')->load_all_reports($record->layout->instance_id);
```

Comment:
Might be worth considering a new method to only return the reports, rather than the reports with the fields. You could even create a new method in `GADS::Layout` such as `$layout->reports`